app/views/handlers/feedback.py

```diff
@@ -59,42 +68,47 @@ def form(self):
             form_data=self._form_data,
         )
 
-    def get_context(self):
+    def get_context(self) -> Mapping[str, Union[str, bool, dict]]:
         return build_feedback_context(self.question_schema, self.form)
 
-    def get_page_title(self):
+    def get_page_title(self) -> Union[str, Any]:
         if self.form.errors:
             return gettext("Error: {page_title}").format(page_title=self.PAGE_TITLE)
         return self.PAGE_TITLE
```

Comment:
The `Any` makes the `str` redundant, the `Any` is just cause of `gettext`, we can make it explicit.
```suggestion
    def get_page_title(self) -> str:
        if self.form.errors:
            title: str = gettext("Error: {page_title}").format(
                page_title=self.PAGE_TITLE
            )
            return title
        return self.PAGE_TITLE
```